src/EditorFeatures/Test/EditAndContinue/EditAndContinueWorkspaceServiceTests.cs

```diff
@@ -233,7 +233,7 @@ private static IEnumerable<string> InspectDiagnostics(ImmutableArray<DiagnosticD
             => actual.Select(d => InspectDiagnostic(d));
 
         private static string InspectDiagnostic(DiagnosticData diagnostic)
-            => $"{(diagnostic.DataLocation != null ? diagnostic.DataLocation.GetFileLinePositionSpan().ToString() : diagnostic.ProjectId.ToString())}: {diagnostic.Severity} {diagnostic.Id}: {diagnostic.Message}";
+            => $"{(string.IsNullOrWhiteSpace(diagnostic.DataLocation.GetFilePath()) ? diagnostic.ProjectId.ToString() : diagnostic.DataLocation.GetFileLinePositionSpan().ToString())}: {diagnostic.Severity} {diagnostic.Id}: {diagnostic.Message}";
```

Comment:
test helper.  just maintaining this behavior for the baselines to stay the same.